docs/get-started/quickstarts/control-motor.md

```diff
@@ -21,30 +21,41 @@ Follow these steps to control your motor:
 
 {{< expand "Step 1: Create a machine" >}}
 
-Go to the Viam app and [add a new machine](/cloud/machines/#add-a-new-machine).
-
-![The 'First Location' page on the Viam app with a new machine name in the New machine field and the Add machine button next to the field highlighted.](/fleet/app-usage/create-machine.png)
+Go to the Viam app.
+Select a location and [add a new machine](/cloud/machines/#add-a-new-machine).
+Click the name of your machine to go to that machine's page.
 
 {{< /expand >}}
+{{%expand "Step 2: Install viam-server" %}}
+
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
+Follow the {{< glossary_tooltip term_id="setup" text="setup instructions" >}} that appear on your new machine's **CONFIGURE** page to install `viam-server` on your computer and connect it to the Viam app.
+
+{{% /expand%}}
+{{< expand "Step 3: Configure a board" >}}
```

Comment:
For reviewer: I'm not sure that we actually need to this just to control the motor, I don't think so? lmk if anyone can confirm and I can remove this step

Reply:
Ohh ok my bad, thanks for checking @JessamyT!